src/components/Timeline.vue

```diff
@@ -162,8 +162,8 @@ export default {
     this.visData.groups = mountVisData(this, 'groups');
     timeline = new Timeline(container, this.visData.items, this.visData.groups, this.options);
 
-    this.events.forEach(eventName =>
-      timeline.on(eventName, props => this.$emit(eventName, props))
+    this.events.forEach(  eventName =>
```

Comment:
Is eslint allowing space before arrow function ?

Reply:
🤔 Doesn't look like it: https://github.com/airbnb/javascript#arrow-functions